src/ngraph/frontend/onnx_import/op/softplus.cpp

```diff
@@ -34,34 +30,37 @@ namespace ngraph
             {
                 NodeVector softplus(const Node& node)
                 {
-                    auto data = node.get_ng_inputs().at(0);
+                    const auto data = node.get_ng_inputs().at(0);
 
-                    std::shared_ptr<ngraph::Node> zero_node =
+                    const std::shared_ptr<ngraph::Node> zero_node =
                         std::make_shared<default_opset::Constant>(
                             data->get_element_type(), data->get_shape(), std::vector<float>{0.f});
-                    std::shared_ptr<ngraph::Node> one_node =
+                    const std::shared_ptr<ngraph::Node> one_node =
                         std::make_shared<default_opset::Constant>(
                             data->get_element_type(), data->get_shape(), std::vector<float>{1.f});
 
-                    std::shared_ptr<ngraph::Node> positive_val_node =
-                        data + std::make_shared<default_opset::Log>(
-                                   std::make_shared<default_opset::Exp>(
-                                       std::make_shared<default_opset::Negative>(data)) +
-                                   one_node);
+                    const std::shared_ptr<ngraph::Node> positive_val_node =
```

Comment:
This is getting hard to read. Please add a comment with math formula produced by the code below.